ta/pkcs11/src/pkcs11_token.h

```diff
@@ -141,6 +141,7 @@ enum pkcs11_proc_state {
  * @tee_hash_op_handle - handle on active hashing crypto operation or
  * TEE_HANDLE_NULL
  * @extra_ctx - context for the active processing
+ * @last_block_size - size of the last block for padding purpose.
```

Comment:
This field should be allocated and referenced using `extra_ctx` field.
A block could also be allocated, to store padded bytes, so that it.s allocated from init and won't fail at final stage due to memory exhaustion.